app/src/main/AndroidManifest.xml

```diff
@@ -6,15 +6,16 @@
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:supportsRtl="true"
-        android:theme="@style/AppTheme">
```

Comment:
This line was taken out mostly because each grouping of activities are going to have a custom theme. We don't really have an overarching theme in this application. So having this here doesn't add any value.